packages/core/src/browser/tree/tree-widget.tsx

```diff
@@ -335,8 +334,8 @@ export class TreeWidget extends ReactWidget implements StatefulWidget {
      */
     protected forceUpdate({ resize }: TreeWidget.ForceUpdateOptions = { resize: true }): void {
         if (this.view && this.view.list) {
+            this.view.list.scrollToPosition(this.node.scrollTop);
```

Comment:
Could elaborate why it should be done here? This method responsible for triggering rerendering. Why should it scroll each time when someone wants to rerender?